lib/src/alloc.h

```diff
@@ -42,7 +42,20 @@ static inline bool ts_toggle_allocation_recording(bool value) {
   return false;
 }
 
-static inline void *ts_malloc(size_t size) {
+#ifndef ts_malloc
+#define ts_malloc(_sz)       ts_malloc_dflt(_sz)
+#endif
+#ifndef ts_calloc
+#define ts_calloc(_cnt,_sz)  ts_calloc_dflt(_cnt,_sz)
+#endif
+#ifndef ts_realloc
+#define ts_realloc(_ptr,_sz) ts_realloc_dflt(_ptr,_sz)
+#endif
+#ifndef ts_free
+#define ts_free(_ptr)        ts_free_dflt(_ptr)
+#endif
+
+static inline void *ts_malloc_dflt(size_t size) {
```

Comment:
I'd prefer to just spell out "default" fully. Can you change these functions to `*_default`?

Reply:
side note: i assume github actions runs through all the necessary tests? lmk if that's incorrect/docs on running any integration/build tests